recipes/kaitai_struct_cpp_stl_runtime/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "cb09b3a84c7e905c6a3fecfe9617cf864d9af781":
```

```suggestion
  "cb09b3a84c7e905c6a3fecfe9617cf864d9af781":
```

Use the official version 0.9. Or, ask to the author for a new tag, 0.8 is from 2020. Or, use the date, not the commit: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases